Advanced function usage/task3.js

```diff
@@ -0,0 +1,49 @@
+function divide (numerator, denominator){
+    if (denominator === 0 || denominator === null) {
+        throw new Error ("Cannot divide by zero");
+    } else if (isNaN (numerator) || isNaN (denominator)) {
```

Comment:
Перше - ти не прибрав else 
```suggestion 
}
if (isNaN (numerator) || isNaN (denominator)) {
```
Друге - ти проігнорував коментар, який я писав про isNaN. Цей метод тут не підходить. Використай typeof

Reply:
Я ж додав || denominator === null, щоб далі можно було через isNaN